packages/cli/lib/errorHandlers/index.js

```diff
@@ -0,0 +1,86 @@
+const { logger } = require('@hubspot/local-dev-lib/logger');
+const {
+  isHubSpotHttpError,
+  isSystemError,
+  isFileSystemError,
+} = require('@hubspot/local-dev-lib/errors/index');
+const { shouldSuppressError } = require('./suppressError');
+const { i18n } = require('../lang');
+const util = require('node:util');
+
+const i18nKey = 'lib.errorHandlers.index';
+
+function logError(error, context = {}) {
```

Comment:
This is now the singular `logError` method that will handle the logging.  So we can call this and let it determine how to log the output.